test/hmactest.c

```diff
@@ -133,6 +133,9 @@ static int test_hmac_run(void)
     int ret = 0;
 
     ctx = HMAC_CTX_new();
+    if (ctx == NULL)
```

Comment:
This should be:
```c
if (!TEST_ptr(ctx = HMAC_CTX_new()))
    return 0;
```

This produces an error message in the logs letting us know what the failure was.